ambry-rest/src/test/java/com.github.ambry.rest/NettyResponseChannelTest.java

```diff
@@ -392,6 +392,23 @@ public void keepAliveTest() {
     channel.close();
   }
 
+  /**
+   * Tests that the underlying network channel is closed when {@link NettyResponseChannel#close()} is called.
+   */
+  @Test
+  public void closeTest() {
+    // request is keep-alive by default.
+    HttpRequest request = createRequestWithHeaders(HttpMethod.GET, TestingUri.Close.toString());
+    EmbeddedChannel channel = createEmbeddedChannel();
+    channel.writeInbound(request);
+
+    // drain the channel of content.
+    while (channel.readOutbound() != null) {
+      ;
```

Comment:
Ah, that's annoying.  I think if you write it as 
```
while (...) {}
```
the formatter will just change it to 
```
while (...) {
}
```
which is slightly better, I guess.